README.md

```diff
@@ -105,11 +105,18 @@ for polymorphic configuration - e.g. `ReconnectionPolicyFactory`. There are also
 that the major configuration options are parseable. To find examples of particular config variants, take a look at the
 [test resources](src/test/resources) folder.
 
+It's possible now to specify an optional `contactPointsType` property in order to provide a contact point that resolves 
+to multiple `InetAddress`es. In order to enable such behaviour, set the `contactPointsType` to `DNS`. Not providing a
+configuration value for this property will keep the existing behaviour - one `InetAddress` per `contactPoints` entry. Please
+note that when `contactPointsType` is set to `DNS`, only `contactPoints` containing exactly one element will be accepted. 
```

Comment:
I'm not sure why we need to limit this to only one element. Is there a reason why we can't resolve all A records for each item in the list (via `InetAddress.getAllByName()`) and then concatenate those arrays to produce the result?

This seems more flexible and would allow scenarios where multiple DNS names are used where each might resolve to multiple addresses.

Hypothetical example: you have a cluster spanning multiple data centres, where each DC has a single DNS name resolving to multiple A records. Config could then be something like this:

```yaml
contactPointType: DNS
contactPoints: ["cass-dc1.corp.com", "cass-dc2.corp.com"]
```

... which might then resolve to:

`contactPoints = [10.0.1.1, 10.0.1.2, 10.0.2.1, 10.0.2.2]`